sync/class.jetpack-sync-module-posts.php

```diff
@@ -85,6 +85,25 @@ function filter_blacklisted_post_types( $args ) {
 		return $args;
 	}
 
+	function remove_embed() {
+		global $wp_embed;
+		remove_filter( 'the_content', array( $wp_embed, 'run_shortcode' ), 8 );
+		// Shortcode placeholder for strip_shortcodes()
+		remove_shortcode( 'embed', array( $wp_embed, 'shortcode' ) );
+		remove_shortcode( 'embed', '__return_false' );
```

Comment:
I though I needed to remove different types of shortcodes. But I only need to remove the embed key. 
`remove_shortcode()` only take one parameter. This should be just `remove_shortcode( 'embed' );